### PR TITLE
Introduce org uuid

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**
# What this PR does
-  Introduce org uuid to ensure we have some unique identifier between clusters for routing chatops requests
- Temporary remove autoconnect personal telegram account - it doesn't work since the verification code became too long. It will be restored later ( Approved by @Matvey-Kuk )

### CODE REVIEW

File: engine/apps/telegram/models/verification/channel.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def is_active(self) -> bool:
 
     @property
     def uuid_with_org_id(self) -> str:
```
Comment (joeyorlando) [inline engine/apps/telegram/models/verification/channel.py:27]: nitpick; maybe rename this property to `uuid_with_org_uuid`?

File: engine/apps/telegram/models/verification/personal.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def is_active(self) -> bool:
 
     @property
     def uuid_with_org_id(self) -> str:
```
Comment (joeyorlando) [inline engine/apps/telegram/models/verification/personal.py:25]: same here

File: engine/apps/telegram/renderers/keyboard.py [hunk 3]
```diff
@@ -83,10 +83,10 @@ def _render_button(self, text: str, action: Action, action_data: Optional[Union[
         callback_data_args = [self.alert_group.pk, action.value]
         if action_data is not None:
             callback_data_args.append(action_data)
-        # Add org id with 'x-oncall-org-id' prefix to callback data.
+        # Add org id with 'oncall-uuid' prefix to callback data.
         # It's a workaroung to pass org_id to the oncall-gateway while proxying requests.
         # TODO: switch to json str instead of ':' separated string.
```
Comment (joeyorlando) [inline engine/apps/telegram/renderers/keyboard.py:88]: nitpick; is this last TODO here still relevant?

File: engine/common/oncall_gateway/oncall_gateway_client.py [hunk 4]
```diff
@@ -71,8 +71,8 @@ def post_slack_connector(self, slack_id: str, backend: str) -> tuple[SlackConnec
         response = self._post(url=self._slack_connectors_url, json=d)
         response_data = response.json()
         return (
-            OnCallConnector(
-                response_data["oncall_org_id"],
```
Comment (joeyorlando) [inline engine/common/oncall_gateway/oncall_gateway_client.py:75]: what is this change related to?

File: engine/apps/telegram/updates/update_handlers/button_press.py [hunk 5]
```diff
@@ -71,10 +71,16 @@ def _get_action_context(data: str) -> ActionContext:
         action_name = args[1]
         action = Action(action_name)
 
-        action_data = args[2] if len(args) >= 3 and not args[2].startswith("x-oncall-org-id") else None
+        action_data = args[2] if len(args) >= 3 and not args[2].startswith("oncall-uuid") else None
```
Comment (joeyorlando) [inline engine/apps/telegram/updates/update_handlers/button_press.py:74]:
```suggestion
        action_data = args[2] if len(args) >= 3 and not cls._is_oncall_identifier(args[2]) else None
```

**Comment by joeyorlando** [comment]
> Temporary remove autoconnect personal telegram account - it doesn't work since the verification code became too long. It will be restored later

could we add the org uuid as a request header instead of in the verification code itself?

File: engine/common/oncall_gateway/oncall_gateway_client.py (hunk 4 above)
Comment (Konstantinov-Innokentii) [reply engine/common/oncall_gateway/oncall_gateway_client.py:75]: It's just a small bugfix related to the interaction betwenn oncall and oncall-gateway

**Comment by Konstantinov-Innokentii** [comment]
> > Temporary remove autoconnect personal telegram account - it doesn't work since the verification code became too long. It will be restored later
> 
> could we add the org uuid as a request header instead of in the verification code itself?

When doing autoconnect oncall redirects user to telegram with code in query params and when you open telegram it uses code from query param. I think we can not control this telegram behaviour

**Review by joeyorlando — APPROVED** [review]
